src/rules/common/no-path-trailing-slash.ts

```diff
@@ -3,10 +3,11 @@ import { UserContext } from '../../walk';
 
 export const NoPathTrailingSlash: Oas3Rule | Oas2Rule = () => {
   return {
-    PathItem(_path: any, { report, key }: UserContext) {
-      if ((key as string).endsWith('/')) {
+    PathItem(_path: any, { report, key, location }: UserContext) {
+      if ((key as string).endsWith('/') && (key as string).length !== 1) {
```

Comment:
check exactly for `/`, if the path is `a` it should still report

Reply:
> if the path is `a` it should still report
could you clarify plz? although, updated as exact check is definitely more readable